source/playbooks/notifications-and-updates.rst

```diff
@@ -6,5 +6,11 @@ To help you keep track of your runs, tasks, and statuses a daily digest is sent
 Daily digest
 ------------
 
+When you're running playbooks in multiple channels it can be overwhelming. The daily digest is sent once a day, in the morning. It lists the actionable items for each run, as well as any outstanding tasks or status updates required. Select the run name to move to that channel.
+
+The digest is on by default. To turn it off, use the slash command ``/playbook digest``.
```

Comment:
The slash command is `/playbook settings digest off` (admittedly a bit of a mouthful but works the same way as Github's)

Reply:
Done!